app/src/main/java/com/example/wearit/ui/WardrobeScreen.kt

```diff
@@ -110,17 +124,45 @@ fun WardrobeClothesListSection(
 fun WardrobeListOfCategories(
     onCategoryChange: (category: Category) -> Unit,
     listOfCategories: List<Category>,
+    currentCategory: Category,
 ) {
-    Column() {
-        Column {
-            LazyColumn {
-                items(listOfCategories) { category ->
-                    Button(onClick = { onCategoryChange(category) }) {
-                        Text(text = "Go to $category")
-                    }
+    val columnPadding = 7.dp
+    Column {
+
+
+
+        LazyColumn {
+
+            itemsIndexed(listOfCategories) { index, category ->
```

Comment:
Jest jakiś powod ze to musi być itemsIndexed? Bo jak nie to lepiej użyć items

Reply:
nie, itemsindexed bylo potrzebne tylko do przesuniecia kafelka, gdzie to przesuniecie musialo byc przemnozone przez index kazdego kafelka, teraz z tego zrezygnowalismy wiec nie jest potrzebne, zmienilem na items